docs/EN_US/DataPatterns/DataPatternsMods/DataPatternsIntro.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<chapter id="BLAS">
```

Comment:
This ID "BLAS" likely re-used elsewhere, would create a build error. 
Please rename with a unique name that is unlikely to occur elsewhere.